dietpi/dietpi-cloudshell

```diff
@@ -1066,6 +1066,9 @@
 
 	}
 
+	# Tor Relay: Include refresh rate delay and kill the monitor process afterwards
+	Update_Scene_9(){ nyx & sleep $REFRESH_RATE; killall -w nyx; }
```

Comment:
Hmm, this is not yet optimal. When exiting (ctrl+c) DietPi-CloudShell while this scene is active, nyx stays running in background but printing to screen. This could be resolved by adding the `killall -w nyx` to the scripts exit trap.

Another thing: User might want to enable this as only scene, but it is killed and re-launched on refresh. Since it is not a static print but a self-refreshing monitor, it is a bid a special case. Probably we can move the kill to the start of the switcher loop (before or after the Update_Banner call): If the current scene is not the Tor monitor, kill nyx. Although not sure about this blank screen thing, needs to be reviewed/tested.
`nyx` must then only be called if it's not running yet. Since ps/pgrep methods are quite slow (recognisable delay), a flag variable would be best.

Reply:
Running `time pgrep "some process"` returns at most 0.03 seconds, on both my VM and RPi.

Reply:
Ah yeah sorry I forgot that this is a specific issue on Debian Bullseye:
```
2020-12-02 17:02:54 root@micha:/var/log# time pgrep mariadb
5162

real    0m0.322s
user    0m0.275s
sys     0m0.038s
```
I should report this to Debian bug tracker, as until Debian Buster it takes a fraction of that indeed. However, always good to avoid external binaries, where possible, especially when the positive result is required for an important step, like killing the background process here.

Reply:
`nyx` is working, but the graph glitches after a few seconds, sending blocks everywhere.